src/views/Languages/comments.graphql

```diff
@@ -0,0 +1,5 @@
+query comments($id: Int!){
+    comments(id: $id){
+        text
+    }
+}
```

Comment:
This seems to be a duplicate of `src/views/Project/comments.graphql`. Move this file to `src/views` and import it from there for both views.